manager/report_handler.py

```diff
@@ -191,9 +188,9 @@ def handle_get(cls, **kwargs):
                                                      SystemPlatform.advisor_evaluated.is_null(False)) &
                                                     (fn.COALESCE(SystemPlatform.host_type, 'null') != HostType.EDGE))
                            .where(SystemVulnerabilities.rh_account_id == rh_account)
-                           .where((SystemVulnerabilities.mitigation_reason.is_null(True)) &
+                           .where((SystemVulnerabilities.when_mitigated.is_null(True)) &
```

Comment:
This query should be left unchanged, since we only need rule hits and that is being selected. Please revert